ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -58,4 +60,187 @@ - (void)testShouldIgnoreVnodeType {
     XCTAssertFalse(ShouldIgnoreVnodeType(static_cast<vtype>(1000), testVnode.get()));
 }
 
+- (void)testFileFlaggedInRoot {
+    bool fileFlaggedInRoot;
+    shared_ptr<mount> testMount = mount::Create();
+    shared_ptr<vnode> testVnode = vnode::Create(testMount, "/foo");
+    
+    testVnode->attrValues.va_flags = FileFlags_IsInVirtualizationRoot;
+    XCTAssertTrue(TryGetFileIsFlaggedAsInRoot(testVnode.get(), NULL, &fileFlaggedInRoot));
+    XCTAssertTrue(fileFlaggedInRoot);
+    
+    testVnode->attrValues.va_flags = FileFlags_IsEmpty;
+    XCTAssertTrue(TryGetFileIsFlaggedAsInRoot(testVnode.get(), NULL, &fileFlaggedInRoot));
+    XCTAssertFalse(fileFlaggedInRoot);
+    
+    testVnode->attrValues.va_flags = FileFlags_Invalid;
+    XCTAssertTrue(TryGetFileIsFlaggedAsInRoot(testVnode.get(), NULL, &fileFlaggedInRoot));
+    XCTAssertFalse(fileFlaggedInRoot);
+    
+    testVnode->attrValues.va_flags = 0x00000100;
```

Comment:
Does `0x00000100` have special meaning? Or was it just something other than `FileFlags_IsInVirtualizationRoot`?

Reply:
Just something different